include/gsl/multi_span

```diff
@@ -2019,96 +2058,90 @@ private:
 
     pointer data_;
     const Span* m_validator;
-    void validateThis() const
-    {
+
+    GSL_SUPPRESS(bounds.1) // NO-FORMAT: attribute
+    void validateThis() const {
```

Comment:
"{" in the next line would be more consistent with the rest of the code.

Reply:
will try to keep revert it, will see if clang-format will change it again